samza-kafka/src/main/scala/org/apache/samza/system/kafka/KafkaSystemFactory.scala

```diff
@@ -71,6 +74,7 @@ class KafkaSystemFactory extends SystemFactory with Logging {
     // Unlike consumer, no need to use encoders here, since they come for free
     // inside the producer configs. Kafka's producer will handle all of this
     // for us.
+    info("Creating kafka producer for system %s, idPrefix %s" format(systemName, clientId))
```

Comment:
Would be better to rename idPrefix with `consumerClientId`. idPrefix doesn't share if it's part of consumerId  or producerId or some other uniqueID in samza.